services/web/server/src/simcore_service_webserver/login/utils.py

```diff
@@ -124,27 +124,35 @@ async def send_mail(app: web.Application, msg: MIMEText):
     smtp_args = dict(
         hostname=cfg.SMTP_HOST,
         port=cfg.SMTP_PORT,
-        use_tls=cfg.SMTP_TLS_ENABLED,
+        use_tls=cfg.SMTP_PROTOCOL == "TLS",
+        start_tls=cfg.SMTP_PROTOCOL == "STARTTLS",
```

Comment:
I would expect something like this here.
```suggestion
        use_tls=cfg.SMTP_PROTOCOL == EmailProtocol.TLS,
        start_tls=cfg.SMTP_PROTOCOL == EmailProtocol.STARTTLS,
```